sass/_moj-overrides.scss

```diff
@@ -1,3 +1,35 @@
 .moj-pagination__results {
   margin: 0;
 }
+
+.moj-filter__selected {
+  background-color: govuk-colour("white");
+  h3 {
+    margin-right: 20px;
+    display: inline-block;
+  }
+
+  .moj-filter-tags {
```

Comment:
shenanigans to ensure the heading and tags are on the same line, but break on next category (sadly the html - which we have no control off - does not model this so the css nonsense)